runtime/j9vm/java9vmi.c

```diff
@@ -1797,4 +1797,19 @@ JVM_BeforeHalt()
 {
 	/* To be implemented via https://github.com/eclipse/openj9/issues/1459 */
 }
+JNIEXPORT jclass JNICALL
+JVM_GetNestHost(JNIEnv *env, jclass clz)
+{
+	assert(!"JVM_GetNestHost unimplemented");
```

Comment:
These methods should still have a return statement; otherwise compilers may issue a warning which may prove fatal.

Reply:
Agreed in principle. But it is not fatal in the build.
```
gcc -DJAVA_SPEC_VERSION=9 -O3 -fno-strict-aliasing -g -MMD -DLINUX -D_REENTRANT -D_FILE_OFFSET_BITS=64 -fPIC -DIPv6_FUNCTION_SUPPORT -DJ9HAMMER -m64 -Wimplicit -Wreturn-type -Werror -I. -I../include -I../oti -I../harmony/include -I../omr/gc/include -I../gc_glue_java -I../hyvm -I../makelib -I../j9vm -I../j9vm -I../j9vm -I../j9vm -I../j9vm -I../j9vm -I../j9vm -I../j9vm -I../nls -I../omr/include_core  -DJ9VM_JAVA9_BUILD=156 -DJ9VM_JCL_SE11  -DUT_DIRECT_TRACE_REGISTRATION -DTR_HOST_X86 -c -o java9vmi.o ../j9vm/java9vmi.c
```
Probably because `-Werror=return-type` is not specified.

Reply:
It could fail on other platforms, such as Windows.

Reply:
Added `return` clauses, and renamed `java9vmi.c` to `java11vmi.c`. Pls have another look.